### ISSUE TITLE
[BUG] expose plugin sends special workspaces to workspace 1

### ISSUE THREAD

**Issue by axelKeizoStahl** [hyprland-community/pyprland#119]
**Pyprland version**
Which version did you use? (copy & paste the string returned by `pypr version`)
2.4.0-8
**Describe the bug**
If you use special workspaces in expose, when closing the "expose" workspace, the special workspaces get sent to  workspace 1.

**To Reproduce**
1. Set "include_special" to true for the expose plugin.
2. Toggle expose to on when a special workspace is open (not focused).
3. Toggle off expose.

**Expected behavior**
The special workspaces should be sent to their original special workspace

**Configuration (provide following files/samples when relevant):**
- pyprland.toml
```
[pyprland]
plugins = [
    "expose"
]

[expose]
include_special = true
```

**Comment by fdev31** [comment]
All good in git ? :)

**Comment by fdev31** [comment]
Fixed, check #120  for details
